puppersim/reacher/reacher_manual_control.py

```diff
@@ -0,0 +1,106 @@
+import reacher_kinematics
```

Comment:
Change to `from puppersim.reacher import reacher_kinematics` and add a empty __init__.py to the reacher folder

Reply:
When importing files inside of a python package, you should always import from the base directory

Reply:
Otherwise your code won't run unless you do `python blah.py` from the right directory

Reply:
ok got it, didn't know that thanks!